fuel-p2p/src/service.rs

```diff
@@ -28,7 +30,7 @@ pub struct FuelP2PService {
     /// Store the local peer id
     pub local_peer_id: PeerId,
     /// Swarm handler for FuelBehaviour
-    swarm: Swarm<FuelBehaviour>,
+    swarm: Swarm<FuelBehaviour<BincodeCodec>>,
```

Comment:
This is much better and way easier to ensure all aspects of the networking is using the same codec.